walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -42,6 +44,36 @@
     timeOptions = getPossibleTimes(occupiedTime, request);
     mandatoryOptions = getPossibleTimes(mandatoryTime, request);
     // Check if the returned times work for all mandatory and optional attendees
+    if (timeOptions.isEmpty() && !mandatoryOptions.isEmpty() && request.getOptionalAttendees().size() > 1) {  
+      List<TimeRange> mandatoryTime2 = new ArrayList<TimeRange>();
```

Comment:
It might be good to give this a more descriptive name, rather than simply appending 2 -- perhaps mandatoryTimeCopy?

Reply:
Updated in commit ad8364a